trade-finance-manager-ui/server/api.get-deals.test.js

```diff
@@ -0,0 +1,75 @@
+const axios = require('axios');
```

Comment:
This is just split from the `api.test.js` file as it was far too big. No changes.